### PR TITLE
Refactor function format in vm

### PR DESCRIPTION
**Pull request by hachi8833**
Updated the test and I'd try again.

I hope this does not disturb your team again.

Thank you,

### CODE REVIEW

File: vm/array.go [hunk 1]
```diff
@@ -77,7 +77,7 @@ func initializeArray(elements []Object) *ArrayObject {
 	return &ArrayObject{Elements: elements, Class: arrayClass}
 }
 
-func init() {
+func initArray() {
```
Comment (st0012) [inline vm/array.go]: The init method is Go's built in method that will be executed when the package is imported.

**Comment by st0012** [comment]
Wow this is a huge work, thanks! But we currently still under rapid development, so large refactoring is too early, we should do this after the core functionality is stable. I'm closing it for now, sorry didn't make it clear earlier 😢

File: vm/array.go (hunk 1 above)
Comment (hachi8833) [inline vm/array.go]: OK, I understood that using `init()` is intentional 😺 
I just thought it looked inconsistent with other initialization functions.

**Comment by hachi8833** [comment]
No problem, I understood your situation 😺 
I just thought it might be good to reduce the repetitive codes in early stage.
I'd do that in a good chance some day!

**Pull request closed without merging**